modules/c++/tiff/include/tiff/Header.h

```diff
@@ -57,14 +57,25 @@ class Header : public io::Serializable
         mId(id), mIFDOffset(ifdOffset)
     {
         bool isBigEndian = sys::isBigEndianSystem();
+        // compilers are now very aggressive about checking propoer strncpy() usage
```

Comment:
Typo in "proper", though I'm not sure this needs to be included since all the `strncpy` were removed.

Reply:
👍 fixed (and added a few more notes)